Rasa X annotations (#532)
RasaHQ/rasa-demo: data/nlu/nlu.md

- whats the diference
- whats the difference between [core](product) and [nlu](product)
- does the [open source](product) version have [core](product)?

> - What are the two components that make up Rasa Open Source?

Reviewer: ```suggestion
- What are the two components that make up [Rasa Open Source](product)?
```

- the difference between [Rasa](product) and [Rasa X](product)
- tell me difference between [Rasa](product) and [Rasa X](product)
- difference between [Rasa](product) and [Rasa X](product)

> - install rasa x with or wothout rasa open source?
> - relationship between rasa open and rasa x

Reviewer: ```suggestion
- install [rasa x](product) with or without [rasa open source](product) ?
- relationship between [rasa open source](product) and [rasa x](product)
```

- thanks for your information
- thanks f

> ## intent:out_of_scope
> - Can I ask you about gardening?
> - sudo poweroff
> 
> ## intent:faq
> - the difference between [rasa](product) and [rasa x]{"entity": "product", "value": "x"}
> 

Reviewer: ```suggestion
```
